lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -316,17 +335,28 @@ class FontSize extends Gatherer {
     passContext.driver.on('CSS.styleSheetAdded', onStylesheetAdd);
 
     await Promise.all([
+      passContext.driver.sendCommand('DOMSnapshot.enable'),
       passContext.driver.sendCommand('DOM.enable'),
       passContext.driver.sendCommand('CSS.enable'),
     ]);
 
+    // We need to find all TextNodes that do not have legible text. DOMSnapshot.captureSnapshot is the
+    // fastest way to get the computed styles of every Node. Bonus, it allows for whitelisting properties.
+    // Once a bad TextNode is identified, its parent Node is needed. DOMSnapshot.captureSnapshot doesn't
+    // give the entire Node object, so DOM.getFlattenedDocument is used. The only connection between a snapshot
+    // Node and an actual Protocol Node is backendId, so that is used to join the two data structures.
+    const snapshot = await passContext.driver.sendCommand('DOMSnapshot.captureSnapshot', {
+      computedStyles: ['font-size'],
+    });
+    // backendIdsToPartialFontData will include all Nodes,
+    const backendIdsToPartialFontData = this.calculateBackendIdsToPartialFontData(snapshot);
+    // but nodes will only contain the Body node and its descendants.
+    const nodes = await getAllNodesFromBody(passContext.driver);
```

Comment:
maybe we now want to differentiate `nodes` from the nodes in the snapshot, somehow? I'm not sure of a great way. `crdpNodes` or something?